src/main/java/org/jabref/gui/keyboard/KeyBinding.java

```diff
@@ -3,6 +3,21 @@
 import org.jabref.logic.l10n.Localization;
 
 public enum KeyBinding {
+    EMACS_DELETE("Emacs delete", Localization.lang("Delete text"), "ctrl+D", KeyBindingCategory.EDIT),
```

Comment:
Please add a new category `KeyBindingCategory.EMACS`. Otherwise, it will confuse users, because the keybindings in the normal menu make the impression that they are always enabled (which they are not)

![grafik](https://user-images.githubusercontent.com/1366654/75486124-43810f80-59ac-11ea-9859-864af3374779.png)